angular/projects/spark-angular/src/lib/components/sprk-masthead/sprk-masthead.component.ts

```diff
@@ -171,28 +171,17 @@ import {
         role="navigation"
         [id]="narrowNavId"
       >
-        <sprk-dropdown
```

Comment:
Let's update the other `<sprk-dropdown>` in this component to use the updated input names. Have we checked the other components to see if they are using `<sprk-dropdown>`? If so, let's update those too.

Reply:
Dropdown is the only component using dropdown, so fixing it there resolves this!